data/pathfinder/paizo/campaign_setting/inner_sea_gods/isg__datacontrols.lst

```diff
@@ -2,3 +2,11 @@
 
 # Name			Dataformat		Visible	Explanation
 FACTSETDEF:SPELL|Deity	DATAFORMAT:String	VISIBLE:YES	EXPLANATION:Just for Listing deities that are predisposed to certain spells.
+#
+FACTSETDEF:DEITY|Symbol	DATAFORMAT:String	VISIBLE:YES	EXPLANATION:The deity's holy symbol
```

Comment:
Uh, Symbol is already made. And it is FACTDEF:Symbol. Hence your error. You cannot have a FACT and FACTSET share the same namespace. A Deity only has one symbol. If the item will be single it should be a FACT, if you can have multiple, then it is a Set of Facts FACTSET.

Cheers,